src/contentScript.js

```diff
@@ -9,54 +14,62 @@ window.addEventListener('SigningRequest', (event) => {
 
 window.addEventListener('GetFCTAddresses', (event) => {
   chrome.storage.sync.get(['fctAddressesRequestWhitelistedDomains', 'fctAddresses'], function(result) {
-    let fctAddressesEvent = new CustomEvent('FCTAddresses', {
-      detail: { success: true, fctAddresses: result.fctAddresses }
-    });
-
-    const requestOrigin = event.target.origin;
-    const whitelistedDomains = result.fctAddressesRequestWhitelistedDomains;
-    if (whitelistedDomains !== undefined && whitelistedDomains.includes(requestOrigin)) {
-      window.dispatchEvent(fctAddressesEvent);
-      addFCTAddressesChangesListener();
+    if (result.fctAddresses === undefined) {
```

Comment:
Let's add a comment here, explaining that in this case there should be no vault created by the user